### PR TITLE
improve Importer documentation 

### PR DESCRIPTION
**Pull request by jmariomejiap**

### CODE REVIEW

**Comment by jmariomejiap** [comment]
Hi @prees1, we are giving the importer guide a pass. I intend to add more clarity to the fields each section uses/needs and to highlight important fields used to create dependency mapping. 
When you get a chance, can you please take a look and help fix anything I might be getting wrong?

File: pages/user guides/importing_to_excel.md [hunk 1]
```diff
@@ -34,42 +34,133 @@ Ie. If your Application has a dependency on a server, you need to have that serv
...
 ### Importing Virtualization Clusters
 
-Tidal Migration uses the following fields to map a virtualization cluster to other resources: _Hypervisor Technology, Servers_. For this field, you must select the property that the column will correspond to, generally this is _Name_.
+Default fields:
+- Hypervisor Technology
+- Servers
+
+There is no required field when importing Virtualization clusters. 
+
+* _Hypervisor Technology_, this field is used to list the technologies you have in your inventory.
```
Comment (prees1) [inline pages/user guides/importing_to_excel.md:69]:
```suggestion
* _Hypervisor Technology_, this specifies the technology that is used to run the hypervisor. Ex. VMware ESXi, Hyper-V. 
```

File: pages/user guides/importing_to_excel.md [hunk 2]
```diff
@@ -34,42 +34,133 @@ Ie. If your Application has a dependency on a server, you need to have that serv
...
+- Name (*)
+- Description
+- Database Size (MB)
+- Database Path
+- Server -> Host Name (*)
+- Environment -> Name (*)
+- Tags
+
+**(*)** Associative fields.
+* _Name_, This field is used to identify and connect a Database to an Application and a Server.
```
Comment (prees1) [inline pages/user guides/importing_to_excel.md:125]: i do understand why _name_ is here, its used to associated the records. Maybe its just me in knowing both the data modelling and external usage, that I find it being here conflicting. When I read it, part of me sees it saying that when importing database records, this field is actively used to identify and connect the database to other records, but it doesn't do that. And internally I know, its not an 'associatian', its a plain old attribute. but maybe calling it associative is best, cause that's is use, except only when importing the other records? I guess really my confusion is this reads the exact same as the one for environment, but they do behave differently, and I can't from this doc alone discern that.

File: pages/user guides/importing_to_excel.md [hunk 3]
```diff
@@ -34,42 +34,133 @@ Ie. If your Application has a dependency on a server, you need to have that serv
...
+* _Cluster -> Host Name_, If a server is a Virtual Server, this field should match the value from the _Servers_ field in the Virtualization Cluster inventory. The **cluster host name** field is be used create the necessary association with its Virtual Cluster and its physical host.
+* _Database Instances -> Name_, If a server is hosting a Database (or multiple), this field should contain the **name**(s) of such database(s).
+* _Environment -> Name_, This field is used to represent the server's environment.
+
+
 
 ### Importing Databases
 
-Tidal Migrations uses the following fields to map a database instance to other resources: _Servers and Environment_. For these fields, you must select the property that the column will correspond to, generally this is _Name_.
+There is no required field when importing Databases.
```
Comment (prees1) [inline pages/user guides/importing_to_excel.md:113]:
```suggestion
There are required fields when importing Databases.
```

File: pages/user guides/importing_to_excel.md [hunk 4]
```diff
@@ -34,42 +34,133 @@ Ie. If your Application has a dependency on a server, you need to have that serv
...
-{% include image.html file="step2.PNG" %}
+    {% include image.html file="step2.PNG" %}
 
-- Click submit to import the data.
+4. Click submit to import the data.
 
-{% include note.html content="If you are importing applications, the name field is required for each application in order to successfully import. For any of the other record types you are importing, there are no fields that are required in order for you to import them." %}
+{% include note.html content="If you are importing applications, the _name_ field is required for each application in order to successfully import. For any of the other record types you are importing, there are no fields that are required in order for you to import them." %}
 
 {% include tip.html content="The Tags option can be used to generate multiple tags and associate them with the record. You can use this track any type of categorical data. Additionally you can search and filter for records based on tags. Multiple tags can be associated based on one column of data, they are parsed and split on any commas (,)" %}
```
Comment (prees1) [inline pages/user guides/importing_to_excel.md:59]: this info:
>Multiple tags can be associated based on one column of data, they are parsed and split on any commas (,)

is probably useful to know all the different attributes that it is true for. ex URLs

File: pages/user guides/importing_to_excel.md [hunk 5]
```diff
@@ -34,42 +34,133 @@ Ie. If your Application has a dependency on a server, you need to have that serv
...
 
 ### Importing Virtualization Clusters
 
-Tidal Migration uses the following fields to map a virtualization cluster to other resources: _Hypervisor Technology, Servers_. For this field, you must select the property that the column will correspond to, generally this is _Name_.
+Default fields:
+- Hypervisor Technology
+- Servers
+
+There is no required field when importing Virtualization clusters.
```
Comment (prees1) [inline pages/user guides/importing_to_excel.md:67]:
```suggestion
There are no required fields when importing Virtualization clusters. 
```

File: pages/user guides/importing_to_excel.md [hunk 6]
```diff
@@ -34,42 +34,133 @@ Ie. If your Application has a dependency on a server, you need to have that serv
...
-Tidal Migration uses the following fields to map a virtualization cluster to other resources: _Hypervisor Technology, Servers_. For this field, you must select the property that the column will correspond to, generally this is _Name_.
+Default fields:
+- Hypervisor Technology
+- Servers
+
+There is no required field when importing Virtualization clusters. 
+
+* _Hypervisor Technology_, this field is used to list the technologies you have in your inventory. 
+
+* _Servers_, this field is used to create associations as you import the rest of your inventory, in this instance, _Servers_ refers to the server (host name) hosting the hypervisor.
```
Comment (prees1) [inline pages/user guides/importing_to_excel.md:71]: >this field is used to create associations as you import the rest of your inventory

I don't follow what this means.

> _Servers_ refers to the server (host name) hosting the hypervisor.

This is definitely accurate but i think not clear and maybe ambiguous, regarding 'hosting the hypervisor'. as there are other servers that host the control plane of the hypervisor. while these physical vms in terms of usage, host the virtual servers that apart of the cluster, and they do technically also host part of the hypervisor control plane to do that.

> A list of physical servers that are used to run all the virtual machines within the Cluster. The servers for each Cluster should be provided as a list of hostnames that are comma (,) separated.

File: pages/user guides/importing_to_excel.md [hunk 7]
```diff
@@ -34,42 +34,133 @@ Ie. If your Application has a dependency on a server, you need to have that serv
...
+
+    {% include note.html content="Make sure your server inventory (when importing virtual servers) contains a field `Cluster` that points back to the Virtualization Cluster it is part of." %}
+
+
 
 ### Importing Servers
 
-Tidal Migrations uses the following fields to map a server to other resources: _FQDN, IP Addresses, Database Instances, Environment, Cluster_. For these fields, you must select the property that the column will correspond to, generally this is _Name_.
+There is no required field when importing Servers.
```
Comment (prees1) [inline pages/user guides/importing_to_excel.md:79]:
```suggestion
There are no required fields when importing Servers. 
```

**Comment by prees1** [comment]
Yea great stuff Mario! Super useful to have this documented. I left some comments on some things for you to consider there.

File: pages/user guides/importing_to_excel.md (hunk 6 above)
Comment (prees1) [reply pages/user guides/importing_to_excel.md:71]: >this field is used to create associations as you import the rest of your inventory

It took me some time but I get what this first part means now. The part that i was struggling with was _the rest of your inventory. I didn't follow what that referred to.

>As you import other records, (ex. Applications, Databases) this field is used to find the correct Server and create those dependencies.

That's just an example of what, in my case, would have avoided me not knowing at first.

**Comment by jmariomejiap** [comment]
> Yea great stuff Mario! Super useful to have this documented. I left some comments on some things for you to consider there.

Thanks a lot, Phil, Super useful. 
Yeah, there are some parts that are a bit tricky to explain.  I'll continue working on your feedback. 👍

File: pages/user guides/importing_to_excel.md (hunk 2 above)
Comment (jmariomejiap) [reply pages/user guides/importing_to_excel.md:125]: Yeah, I see what you mean, @prees1. Maybe one of the issues is the word "connect". 

What about something like this:
```
_Name_, This field is used to identify your Database and it is helpful when associating a Database with an Application and a Server.
```

File: pages/user guides/importing_to_excel.md (hunk 6 above)
Comment (jmariomejiap) [reply pages/user guides/importing_to_excel.md:71]: Nice, your last suggestion reads a lot better and reduces the extra complexity

File: pages/user guides/importing_to_excel.md (hunk 4 above)
Comment (jmariomejiap) [reply pages/user guides/importing_to_excel.md:59]: Do you think we should investigate ☝️ now before merging this PR. or something for us to do in the future?

File: pages/user guides/importing_to_excel.md (hunk 2 above)
Comment (prees1) [reply pages/user guides/importing_to_excel.md:125]: I get stuck on 'helpful' there, is that helpful for me in someway, if so how? or helpful to someone else, who? or to the software?

I'd take what you have and say why it is helpful _to them_ very clearly:

>_Name_, When you are importing other records, Applications and Servers, and you provide a set of dependent databases with them the _Name_ field for Database Instances is used to identify those databases and associate the databases as dependencies for those other records. This is helpful when you have some existing dependencies, between Databases and Servers or Applications, in your Excel files and want to capture and leverage them.

File: pages/user guides/importing_to_excel.md (hunk 4 above)
Comment (prees1) [reply pages/user guides/importing_to_excel.md:59]: i can't see any reason it is _needed_ in order to make any of the other changes already done here.

File: pages/user guides/importing_to_excel.md (hunk 2 above)
Comment (jmariomejiap) [reply pages/user guides/importing_to_excel.md:125]: Nice. Thanks a lot 👍

**Pull request merged**